cargo-pgrx/src/command/sudo_install.rs

```diff
@@ -0,0 +1,92 @@
+use std::path::{Path, PathBuf};
+use std::process::Command;
+
+use owo_colors::OwoColorize;
+
+use crate::command::package::Package;
+use crate::CommandExecute;
+
+/// Like `cargo pgrx install`, but uses `sudo` to copy the extension files
```

Comment:
I wonder if this would be better as `cargo pgrx install --sudo`. This is fine though.

I also slightly wonder if it's worth warning users who try to `sudo cargo pgrx install`, since it's both a footgun and a big security liability (there are definitely reasons to do it, so "warn" seems right rather than actually forbidding it).

Reply:
Most people won't even be able to because Rust is not installed as the root user. If you try, it will yell at you for...a while and ask you to set all kinds of environment variables before it will allow itself to be installed as root. But yes, I agree, a simple `if whoami == root then exit 1` ini this case would not be a terrible idea.

Thinking out loud, `cargo pgrx install` would, by default never really work on any Linux system because for most people this would resolve to the installation in `/usr` . It works great on Mac because brew installs into `/opt/homebrew` where the user is allowed read/write access.

Reply:
I'll rejigger this so that it is `cargo pgrx install --sudo`.  The spelling of this isn't a hill I'm willing to die on.

If we want a `if whoami == root` check, then we can do that as another PR.  That feels orthogonal to this.